src/components/clusters/ClusterDetailsMultiRegion/components/AccessControl/UsersSection/AddUserDialog.jsx

```diff
@@ -24,13 +24,18 @@ const AddUserDialog = ({
   isAddUserSuccess,
   resetAddUserMutate,
   isROSA,
+  isHypershift,
 }) => {
   const dispatch = useDispatch();
 
   const [selectedGroup, setSelectedGroup] = React.useState('dedicated-admins');
   const [userId, setUserId] = React.useState('');
   const [userIdTouched, setUserIdTouched] = React.useState(false);
 
+  const iamOperatorRolesLink = isHypershift
+    ? links.ROSA_AWS_IAM_OPERATOR_ROLES
+    : links.ROSA_CLASSIC_AWS_IAM_OPERATOR_ROLES;
+
```

Comment:
If you already give a variable to calculate the correct link for Hypershift, should we also include the logic to differentiate between OSD / ROSA ?